lib/ansible/cli/config.py

```diff
@@ -262,11 +266,16 @@ def _list_entries_from_args(self):
         '''
         build a dict with the list requested configs
         '''
-        config_entries = {}
+
         if context.CLIARGS['type'] in ('base', 'all'):
             # this dumps main/common configs
             config_entries = self.config.get_configuration_definitions(ignore_private=True)
 
+            # for base and all, we include galaxy servers
+            config_entries['GALAXY_SERVERS'] = {}
+            for server in self._galaxy_servers:
+                config_entries['GALAXY_SERVERS'][server.upper()] = self.config.get_configuration_definitions('galaxy_server', server)
+
         if context.CLIARGS['type'] != 'base':
             config_entries['PLUGINS'] = {}
```

Comment:
This is what's probably failing in CI — since the var is not declared for all code branches anymore, I bet this is hitting an attempt to access a non-existing identifier...
```console
ERROR! Unexpected Exception, this is probably a bug: cannot access local variable 'config_entries' where it is not associated with a value
```
*(https://dev.azure.com/ansible/ansible/_build/results?buildId=110242&view=logs&j=360b4196-6892-562b-fec3-7c4f7a0166c3&t=3890d6d7-9244-5c94-dad4-549e0d215adb&l=8019)*